modules/gitops-deploy-resources-different-namespaces.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-resources-different-namespaces_{context}"]
+= Deploy resources to a different namespace
+
+[role="_abstract"]
+To allow the Argo CD to manage resources in other namespaces apart from where it is installed, configure the target namespace with a label "argocd.argoproj.io/managed-by".
+
+.Procedure
+. Run the following command to create/configure the namespace.
```

Comment:
```suggestion
. Run the following command to configure the namespace.
```

Reply:
If the namespace should also be created, customer needs to run `oc create namespace <namespace name>` beforehand.